azuredevops/internal/service/taskagent/resource_agentpool.go

```diff
@@ -46,6 +46,11 @@ func ResourceAgentPool() *schema.Resource {
 				Optional: true,
 				Default:  false,
 			},
+			"auto_update": {
```

Comment:
The API testing shows that  `auto_update` can only be set to `true` on create. An update request needs to be called after the creation to set  `auto_update` to `false`